protocol.go

```diff
@@ -39,47 +35,30 @@ func (info ProtocolInfo) Clone() ProtocolInfo {
 
 const (
 	// StreamsFeature represents streams support (supported by connector).
-	StreamsFeature ProtocolFeature = 0
+	StreamsFeature iproto.Feature = 0
 	// TransactionsFeature represents interactive transactions support.
 	// (supported by connector).
-	TransactionsFeature ProtocolFeature = 1
+	TransactionsFeature iproto.Feature = 1
 	// ErrorExtensionFeature represents support of MP_ERROR objects over MessagePack
 	// (supported by connector).
-	ErrorExtensionFeature ProtocolFeature = 2
+	ErrorExtensionFeature iproto.Feature = 2
 	// WatchersFeature represents support of watchers
 	// (supported by connector).
-	WatchersFeature ProtocolFeature = 3
+	WatchersFeature iproto.Feature = 3
 	// PaginationFeature represents support of pagination
 	// (supported by connector).
-	PaginationFeature ProtocolFeature = 4
+	PaginationFeature iproto.Feature = 4
+	// WatchOnceFeature represents support of WatchOnce request types.
+	WatchOnceFeature iproto.Feature = 6
 )
```

Comment:
What for do we need this constants? We could just use `iproto.IPROTO_FEATURE_BLA` instead of `go-tarantool.BlaFeature`.

Reply:
Updated.